how-to/security_hardening.rst

```diff
@@ -0,0 +1,67 @@
+JAAS: Security Hardening
+========================
+JIMM, the service at the centre of JAAS can be hardened in a number of ways. This 
+document details how you can harden the security of your JAAS deployment. 
+
+CORS
+----
+Cross-Origin Resource Sharing (`CORS <https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS>`__) 
+is an HTTP-header based mechanism that allows a server to indicate any origins (domain, scheme, or port) 
+other than its own from which a browser should permit loading resources. CORS also relies on 
+a mechanism by which browsers make a "preflight" request to the server hosting the cross-origin 
+resource, in order to check that the server will permit the actual request. In that preflight, 
+the browser sends headers that indicate the HTTP method and headers that will be used in the 
+actual request.
+
+To set CORS on JIMM, use the configuration option "cors-allowed-origins".
+
+Ingress TLS
+-----------
+The NGINX Ingress Integrator is a a charm responsible for creating Kubernetes ingress rules, 
```

Comment:
i feel like this section is very "heavy", we should add another doc page with "how to enable TLS for ingress" like we have for livepatch, and here just the link to  it

Reply:
yes, exactly I see this as a good separate HOW-TO.
Happy to approve this PR as it easy but I think we should add a TLS ingress how to

Reply:
i agree, or maybe this links the how to? ill do that instead and make a "TLS" how to